### DISCUSSION TITLE
Tournament Client > round editor > adding map by id fails

### DISCUSSION THREAD

**Discussion by itotallygetthat** [ppy/osu#15914] [Q&A] [answered]
2021.1202.0-lazer
Windows 10

Existing maps already in a saved tournament.json load ok, but entering new ones fail.  

To reproduce: add new map to a pool.  Enter the map id.  If working properly, a valid map id will load map info in a box to the right.   Currently, the map info box is not appearing.

Network log seems to have relevant error messages, other logs don't seem to have anything interesting.
[network.log](https://github.com/ppy/osu/files/7647305/network.log)

(not sure on the discussion category, neither of the 2 available options seem to fit.)

**Comment by peppy** [comment, marked as answer]
Probably the same as https://github.com/ppy/osu/issues/15912#issuecomment-985256658, which was posted minutes ago.

**Reply by itotallygetthat** [reply]
Yep, duplicate, sorry about that.

**Reply by peppy** [reply]
This should be fixed now, was an osu-web side issue.
